test/ui-testing/loan_renewal.js

```diff
@@ -527,8 +525,8 @@ module.exports.test = (uiTestCtx, nightmareX) => {
               .click('#clickable-edit-item')
               .wait('#clickable-delete-entry')
               .click('#clickable-delete-entry')
-              .wait('#clickable-delete-item-confirmation-confirm')
```

Comment:
I'm still not sure why this is changing. Is there a corresponding PR in ui-circulation that needs to be merged first? The new `id` selectors still do not look correct to me.